### DISCUSSION TITLE
Technical documentation for Noah-MP v4.0.1

### DISCUSSION THREAD

**Discussion by martina01natali** [NASA-LIS/LISF#1600] [Q&A] [answered]
Hi community,
I'm a new user diving into the physics of the model, I have the original technical documentation file from 2011 (https://www.jsg.utexas.edu/noah-mp/files/Noah-MP_Technote_v0.2.pdf) (not sure about the Noah-MP version here but I would guess the 2.7) and of course the v5 (https://github.com/NCAR/noahmp/blob/master/docs/NoahMP_v5_technote.pdf).

I'm using Noah-MP 4.0.1 on LIS, does anyone know if a technical note with nice equations was published for this version? Thanks in advance.

**Comment by dmocko** [comment, marked as answer]
Hi @martina01natali 

No, there is not a similar document for Noah-MP-4.0.1.

However, most of the physics is the same between 4.0.1 and 5.0.  The code was re-factored (new variable names, etc.),
and a couple physics options were added.  But the majority of the equations from the v5 Tech Note also apply to the
4.0.1 version of the code.

By the way, NCAR and NASA are working on adding Noah-MP-5.0 into LIS, and plan on having it available for use in a few months.

**Reply by martina01natali** [reply]
Thanks again, I guessed so... I will refer to the v5 documentation then and check in the code is the equations are missing. Just as a suggestion, it would be helpful to have an indication on the single processes that are "new" in each version, I mean also in future versions of the technical documentation. Thank you for your fast response and also for the heads up on the new version's implementation!
